README.md

```diff
@@ -35,7 +35,7 @@ If you do not wish to run NodeGoat on Heroku, please follow these steps to setup
 
 * Clone the github repository
 ```
-git clone https://github.com/OWASP/NodeGoat.git
+git clone https://github.com/binarymist/NodeGoat.git
```

Comment:
Oh, I just accepted a PR from my friend that is working with me in the workshop in two days time. I only merged this into my fork... I thought. Github must decide it needs to go upstream automatically. So no, it should only be in my fork.